CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 4.6.0 (2021-04-19)
+
+- Updated the `skyux eject` command to create a directory for string resources if one does not exist. [#116](https://github.com/blackbaud/skyux-sdk-cli/pull/116)
+
+- Updated the `skyux eject` command to preserve 3rd-party dependencies in `package.json`. [#117](https://github.com/blackbaud/skyux-sdk-cli/pull/117)
```

Comment:
```suggestion
- Updated the `skyux eject` command to preserve third-party dependencies in `package.json`. [#117](https://github.com/blackbaud/skyux-sdk-cli/pull/117)
```
We usually spell out "third-party." Certainly not a dealbreaker though, so I'll go ahead and approve the PR in case you don't want to make the change and wait for the checks to pass again. If you do make the change, just let me know if I need to re-approve. Thanks.

Reply:
Fixed.